src/resources/shared/openshift/cloud_prepare_aws.md

```diff
@@ -0,0 +1,28 @@
+* You need to extract your infra ID and region in order to use the `subctl cloud prepare` command.
+  Its possible to extract them from the `metadata.json` file that the OpenShift Installer created:
+
+```bash
+metadata_file=<path/to>/metadata.json
+infra_id=$(jq -r .infraID $metadata_file)
+region=$(jq -r .aws.region $metadata_file)
+```
+
+* Run the `subctl cloud prepare aws` command for **cluster-a** and **cluster-b**:
+
+```bash
+subctl cloud prepare aws --kubecontext cluster-a --infraid $infra_id --region $region
```

Comment:
1. It does not look like `--kubecontext` is a valid flag. 
2. It's `infra-id` and not `infraid`
3. Need to add a note/instructions for the user about the missing permissions issue

Reply:
>     1. It does not look like `--kubecontext` is a valid flag

Yeah thats a small bug we need to fix..
https://github.com/submariner-io/cloud-prepare/issues/31

> 
>     2. It's `infra-id` and not `infraid`

Done, thanks

> 
>     3. Need to add a note/instructions for the user about the missing permissions issue

Also done :)